src/Equinox.MessageDb/MessageDbClient.fs

```diff
@@ -14,15 +14,24 @@ type private Format = ReadOnlyMemory<byte>
 [<Struct>]
 type ExpectedVersion = Any | StreamVersion of int64
 
+module private Queries =
```

Comment:
I dont see any symmetry here which suggests keeping them together adds value
esp for the write_message, you want the code to be beside the query to show the correlation between the order of adding to the list vs the order in the string, no?

Reply:
Main goal here is to make absolutely sure that the string for the query is allocated once and only once. I don't know a good way to do that while also maintaining colocation of the query and parameter application

Reply:
I think I caught your drift, I've colocated the query text and the command preparation